lib/queue.js

```diff
@@ -223,6 +223,91 @@ Queue.prototype.checkHealth = function (cb) {
   return promise;
 };
 
+Queue.prototype._scanForJobs = function(cursor, set, cb) {
+  // Ideally, we would be able to paginate redis sets like we do sorted ranges.
+  // For an MVP, however, we're aiming to replace our usage of Arena for Bull 1.x
+  // and Bull 1.x calls SMEMBERS -- which is more blocking than SSCAN.
+
+  this.client.sscan(
+    this.toKey('failed'),
+    cursor,
+    'COUNT', '100',
```

Comment:
>it doesn't matter here

should still be configurable, not because it changes the size of returned results, but because it determines the load on redis